test/fixtures.yml

```diff
@@ -371,6 +371,11 @@ jetpay:
 komoju:
   login: sk_f1dd75ce3d5cad477eac0c827c1cac8eaa51ede3
 
+latitude19:
```

Comment:
If the secret is test-only (and it's really the only thing that's sensitive), then we can commit.